src/pilot/templates/neutron-ovs-dpdk.yaml

```diff
@@ -52,3 +52,25 @@ parameter_defaults:
   # Configure the classname of the firewall driver to use for implementing
   # security groups.
   NeutronOVSFirewallDriver: openvswitch
+
```

Comment:
I don't think we need any of these changes here because all of this is already included in dell-environment.yaml, which is always included in the deploy command.

Reply:
Removed parameter duplication.